iox_catalog/src/interface.rs

```diff
@@ -448,6 +448,12 @@ pub trait NamespaceRepo: Send + Sync {
 
     /// Gets the namespace by its unique name.
     async fn get_by_name(&mut self, name: &str) -> Result<Option<Namespace>>;
+
+    /// Update the limit on the number of tables that can be created per namespace.
```

Comment:
```suggestion
    /// Update the limit on the number of tables that can exist per namespace.
```

If we ever implement table deletion, I think we'll likely count the number of currently existing tables, not the number of tables that ever existed.